include/vcpkg/base/messages.h

```diff
@@ -998,10 +1075,13 @@ namespace vcpkg
                     (msg::value),
                     "{value} is a sha.",
                     "SHA512's must be 128 hex characters: {value}");
+    DECLARE_MESSAGE(IncorrectArchiveFileSignature, (), "", "Incorrect archive file signature");
+    DECLARE_MESSAGE(IncorrectLibHeaderEnd, (), "", "Incorrect lib header end");
     DECLARE_MESSAGE(IncorrectNumberOfArgs,
                     (msg::command_name, msg::expected, msg::actual),
                     "'{expected}' is the required number of arguments. '{actual}' is the number of arguments provided.",
                     "'{command_name}' requires '{expected}' arguments, but '{actual}' were provided.");
+    DECLARE_MESSAGE(IncorrectPESignature, (), "", "Incorrect PE signature");
```

Comment:
Maybe you should add a comment to explain what `PE` means.

Reply:
I think this is fine; it's the message the old code used. (I can see an argument to add paths to all the messages about malformed PE/COFF files though)